docs/package_info/glossary.rst

```diff
@@ -0,0 +1,46 @@
+Glossary
+========
+
+.. _access_token:
+
+* ``Access Token``: A token that is obtained from authenticating with Reddit.
+  The token lasts for an hour, so multiple tokens may need to be obtained.
+
+.. _camel_case:
+
+* ``Camel Case``: A type of variable notation where multiple words are
+  seperated by upper case. For example, the glossary entry for
+  :ref:`Access Token <access_token>` would be written as ``accessToken`` in
+  Camel Case.
+
+.. _creddit:
+
+* ``Creddit``: Back when the only award was ``Reddit Gold``, a creddit was
+  equal to one month of Reddit Gold. Creddits have been converted to
+  ``Reddit Coins``. See :ref:`this <gild>` for more info about the old Reddit
+  Gold system.
+
+.. _fullname:
+
+* ``Fullname``: The fullname of an object is the object's type followed by an
+  underscore and it's base-36 id.
+
+.. _gild:
+
+* ``Gild``: Back when the only award was ``Reddit Gold``, gilding a post meant
+  awarding one month of Reddit Gold. Currently, gilding means awarding one
+  month of ``Reddit Platinum``, or giving a ``Platinum`` award.
+
+.. _snake_case:
+
+* ``Snake Case``: Another type of variable notation where words are seperated
+  by underscores. For example, the glossary entry for
+  :ref:`Access Token <access_token>` would be written as ``access_token`` in
+  Snake Case.
+
+.. _websocket:
+
+* ``Websocket``: A special connection type that supports both a client and a
+  server (the running program and reddit respectively) sending multiple
+  messages to each other. A websocket may not wait for a client to fully
+  connect before sending messages.
```

Comment:
I'd omit the second sentence and replace it with an explanation of why this is relevant to Reddit (because of the `submit_image` and `submit_video` methods).